py/server/deephaven/table.py

```diff
@@ -319,6 +320,34 @@ def meta_table(self) -> Table:
     def j_object(self) -> jpy.JType:
         return self.j_table
 
+    def get_table_attributes(self) -> Dict[str, Any]:
```

Comment:
Why `get_table_attributes` and `with_table_attributes` instead of `get_attributes` and `with_attributes`? It's a method on a table so the `_table_` part seems redundant

Reply:
Python users are conditioned to access object attributes in a Pythonic way, i.e. through the __dict__ attribute (the built-in vars() function, getattr(), setattr()), __getattr__, __getattribute__, and the more advanced descriptors. Because the name get_attributes() is a bit too generic, so in order to highlight the difference and to avoid questions like 'why do you need get_attributes', I came up with those names without too much conviction. Suggestions for better names are welcome.

Reply:
After looking this over, I agree with Chip's suggestion to use `attributes` here.